### ISSUE TITLE
Promoting IcedFrisby

### ISSUE THREAD

**Issue by paulmelnikow** [IcedFrisby/IcedFrisby#103]
Hiya! Since we’ve a few of us are actively working on this and we’re progressing nicely to a 2.0 API, I’d like to make an effort to promote this library.

There are some more things we can do in terms of features and code, like return promises, be test-runner agnostic, test the plugin support, and tighten up the codebase. Though good features won’t promote the project on its own! Here are some good resources, with lots and lots of ideas:

- https://hacks.mozilla.org/2014/05/open-source-marketing-with-velocityjs/
- https://github.com/LisaDziuba/Marketing-for-Engineers
- https://github.com/rdp/open-source-how-to-popularize-your-project#links

Probably we'll want to plan most of this in conjunction with the 2.0 launch, though I thought we could get the ball rolling.

@Fishbowler Are you in?

**Comment by Fishbowler** [comment]
Yeah! I'd love to start doing more of this, especially as a new contributor. Rather than sitting down and thinking "what shall I do?", I'd prefer to think "what shall I do from this list?"

I've also got some avenues in training within testing community that could help too - there's lots of people who wanna know how to build an API testing framework, so providing an awesome resource on this which just happens to use IcedFrisby could be a win.

**Comment by paulmelnikow** [comment]
Cool!

I was thinking too, that it'd be fun to do a blog post on the framework I wrote for Shields. That's when I found IcedFrisby in the first place. I wanted to design an API testing framework that was easy enough for people to use even if they've never written API tests… or APIs… or tests… or JavaScript.

I [sketched the interface I wanted](https://github.com/paulmelnikow/shields-tests/commit/a9d45420b7f1a03c072bd5933667f6d5377a8ada#diff-5e4a4c62c654cfca2607dcde71e0b3d2) – that was my third draft – and then discovered Frisby, which looked _nearly identical_. It was on hiatus, though, with a big unmerged feature branch that would have added Mocha support, and also lots of unmerged pull requests. I didn't want to switch to Jasmine, either. And that's when I found this!

Also in the long term, we might want to consider merging with Frisby. That project is in a healthier place than it was eight months ago.

**Comment by Fishbowler** [comment]
Seems that Frisby has moved from Jasmine to Mocha to Jest now. Interesting move. Certainly enables some nice parallelism in that runner.
